src/impls.rs

```diff
@@ -210,19 +206,22 @@ impl<T: Config> Pallet<T> {
 			Self::get_data_before(query_id, timestamp).unwrap_or_default();
 		let mut price_change = 0; // price change from last value to current value
 		if feed.details.price_threshold != 0 {
-			let v1 =
-				Self::bytes_to_price(value_retrieved.expect("value retrieved checked above; qed"))?;
-			let v2 = Self::bytes_to_price(value_retrieved_before)?;
-			if v2 == Zero::zero() {
+			let v1 = BytesToU256::convert(
```

Comment:
can we use some specific name for variables v1 & v2 to make it more readable?

Reply:
That's how they are in the source contracts, so prefer to keep them as they are so that Tellor can read the Rust code alongside the Solidity code to follow what is going on.

Reply:
Perhaps a compromise would be a comment(s) about what v1 vs v2 is? Agree that the original source's readability could be improved.

Reply:
Added a comment above the definition of v1/v2. 

@oraclown fine to merge?

Reply:
Looks good to me! Thanks @evilrobot-01